saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -497,13 +499,11 @@ class Meta:
     @transaction.atomic
     def perform_mutation(cls, _root, info, ids, **data):
         _, pks = resolve_global_ids_to_primary_keys(ids, ProductVariant)
-        # get draft order lines for variants
```

Comment:
In other places, you left this comment. In my opinion, we should remove all or don't remove any of this comment.